serverless/sql-databases/how-to/use-row-level-security.mdx

```diff
@@ -0,0 +1,90 @@
+    ```bash
+    psql "postgres://[user-or-application-id]:[api-secret-key]@[database-hostname]:5432/[database-name]?sslmode=require"
+    ```
+
+2. Add sample data to the database using the following command:
+    ```sql
+    CREATE TABLE pets (name varchar, keeper varchar, id int);
+    INSERT INTO pets VALUES ('Stuart','role_admin',1),('Nemo','role_admin',2),('Alfie','role_readwrite',3),('Peanut','role_readwrite',4);
+    ```
+
+3. Run the command below to enable **Row Level Security**:
+    ```sql
+    ALTER TABLE pets ENABLE row level security;
+    ```
+
+4. Run the command below to create a PostgreSQL policy so that users or applications connecting with `role_readwrite` can access a `pet` row only if its `keeper` column value is `role_readwrite`:
+    ```sql
+    CREATE POLICY pets_keeper ON pets TO role_readwrite USING (keeper = current_user);
+    ```
+
+5. (Optional) Run the command below to check that you can see all the data with your current connection:
+    ```sql
+    SELECT * FROM pets;
+    ```
+    All the data contained in the database displays, as you are connected with `role_admin`.
+    
+    <Message type="tip">
+    You can verify the current role your are connected with using the following command:
+    ```sql
+    SELECT current_user;
+    ```
+    </Message>
+
+## Create an IAM application with Row Level Security enabled
+
+1. Create a new [IAM application](/identity-and-access-management/iam/how-to/create-application/).
+
+2. Create a new [IAM policy](/identity-and-access-management/iam/how-to/create-policy/), and add the **ServerlessSQLDatabaseDataReadWrite** permission set to the application you just created.
+
+    <Message type="note">
+    You must provide **ServerlessSQLDatabaseDataReadWrite** permission set and not **ServerlessSQLDatabaseReadWrite** permission set. Indeed, all connections to your database performed with the former permissions set will use `role_readwrite` in PostgreSQL, whereas all connections performed with the latter, or **ServerlessSQLDatabaseFullAccess** will use `role_admin` in PostgreSQL.
+    </Message>
+
+3. Create an [API Key](/identity-and-access-management/iam/how-to/create-api-keys/) for this application, and connect to your Serverless SQL Database with this application.
+    ```bash
+    psql "postgres://[new-application-id]:[new-api-secret-key]@[database-hostname]:5432/[database-name]?sslmode=require"
+    ```
+
+4. Run the following command to list the `pets` this application has access to:
+    ```sql
+    SELECT * FROM pets;
+    ```
+    Only the pets with a `keeper` column value of `role_readwrite` display. Your new application can now only access a specific subset of rows based on its permissions.
+
+    <Message type="tip">
+    Row level security and policies can be created or deleted by a table owner. In this example, you can check table owner with the following command:
```

Comment:
```suggestion
    Row-level security and policies can be created or deleted by a table owner. In this example, you can check the table owner with the following command:
```

Reply:
As Postgres's website uses the hyphen, I'm leaving suggestions to use it everywhere here, cause there were some inconsistencies. Feel free to ignore them if you prefer to keep hyphen-free. :)